draft-ietf-tls-tls13.md

```diff
@@ -2323,102 +2315,138 @@ The "extension_data" field of this extension contains a
 
 %%% Signature Algorithm Extension
        enum {
-           none(0),
-           md5_RESERVED(1),
-           sha1(2),
-           sha224_RESERVED(3),
-           sha256(4), sha384(5), sha512(6),
-           (255)
-       } HashAlgorithm;
+           // RSASSA-PKCS-v1_5 algorithms.
+           rsa_pkcs1_sha1 (0x0101),
+           rsa_pkcs1_sha256 (0x0301),
+           rsa_pkcs1_sha384 (0x0401),
+           rsa_pkcs1_sha512 (0x0601),
```

Comment:
I'm kind of confused by these code point assignments, which seem to not actually match their matching SignatureAndHashAlgorithm values from 1.2.